doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/util/CertificateUtils.kt

```diff
@@ -0,0 +1,66 @@
+package com.doordeck.multiplatform.sdk.util
+
+import com.doordeck.multiplatform.sdk.util.Utils.decodeBase64ToByteArray
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toInstant
+import kotlin.time.Duration.Companion.days
+
+internal fun String.isCertificateAboutToExpire(): Boolean = try {
```

Comment:
Right now, Mac/Windows is using this function to verify the expiration date of the certificate because I have not been able to implement anything better or functional on those platforms.